kafka-bin-scripts/README.adoc

```diff
@@ -31,6 +31,8 @@ As a developer of applications and services, you can use the Kafka bin scripts t
 The Kafka bin scripts are a set of shell scripts that are included with the Apache Kafka distribution.
 With these bin scripts, you can produce and consume messages for your Kafka instances.
 
+NOTE: The command examples in this quick start demonstrate how to use the Kafka bin scripts on Linux and macOS. If you're using Windows, you should use the Windows versions of the bin scripts. For example, instead of the `__<Kafka-distribution-dir>__/bin/kafka-console-producer.sh` script, you would use the `__<Kafka-distribution-dir>__\bin\windows\kafka-console-producer.bat` script.
+
```

Comment:
@bhardesty Trivial, but even with informal tone I think we're still going for minimalism and action-focused language, so I'd still still avoid "should" and "would" unless we have a good reason. I'd just say "If ..., use the Windows ..."  and "script, use the ...". Not critical if you disagree. Just my view.